BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -104,19 +104,31 @@
     <p><%= f.label :containerhub_image_name, "Container image name:" %><br/>
       <%= f.text_field :containerhub_image_name %><br/>
       <div class="field_explanation">
-        The name and tag of the Container image in which the tool is installed,
+        The name and tag of the container image in which the tool is installed,
         for instance "centos:latest". This name refers to the Docker/Singulariry index
         accessed by the Bourreau, which is configured manually in the Bourreau
         for now.
       </div>
     </p>
   </span>
 
+  <span title="Index of the container image">
```

Comment:
The order of the columns should be more intuitieve, the index shoudl be to the left of the image name, since URLS are built like that. This is what it looks like in my browser:

![screen shot 2017-09-19 at 16 26 42](https://user-images.githubusercontent.com/777588/30613894-ac42a268-9d57-11e7-91e5-fb0a11351dc0.png)